src/common/function/FunctionManager.cpp

```diff
@@ -577,10 +582,78 @@ FunctionManager::FunctionManager() {
         }
         case Value::Type::INT:
         case Value::Type::FLOAT: {
-          if (args.size() == 2) {
+          if (args.size() >= 2) {
             if (args[1].get().type() == Value::Type::INT) {
+              auto val = args[0].get().getFloat();
               auto decimal = args[1].get().getInt();
-              return std::round(args[0].get().getFloat() * pow(10, decimal)) / pow(10, decimal);
+              auto factor = pow(10.0, decimal);
+
+              string mode = "half_up";
```

Comment:
Default rounding mode here?
Add test case:
```
round(-1.5, 1)  
-1.0 is expected.
```

Reply:
I guess `round(-1.5, 1) ` should be -1.5.

Reply:
You're right, my mistake.